Code/Sandbox/Editor/CryEdit.cpp

```diff
@@ -590,6 +590,7 @@ class CEditCommandLineInfo
             {{"project-path", "Supplies the path to the project that the Editor should use", "project-path"}, dummyString},
             {{"engine-path", "Supplies the path to the engine", "engine-path"}, dummyString},
             {{"project-cache-path", "Path to the project cache", "project-cache-path"}, dummyString},
+            {{"project-user-path", "Path to the project user path", "user-path"}, dummyString}
```

Comment:
Can you add an entry for `project-log-path` as well?

Reply:
Addressed in latest